pkg/sdk/informer.go

```diff
@@ -40,14 +42,16 @@ type informer struct {
 	namespace           string
 	context             context.Context
 	deletedObjects      map[string]interface{}
+	metrics             *metrics.Collector
```

Comment:
We have this as `collector *metrics.Collector` in sdk/api.go. We should make them consistent. 
Rename it to either collector or both of them to something similar.